src/feinstaub-api.js

```diff
@@ -10,30 +12,35 @@ let api = {
   fetchNow() {
     return fetch(URL).then(response => response.json());
   },
-
   // fetches from /now, ignores non-finedust sensors
   // /now returns data from last 5 minutes, so we group all data by sensorId
   // and compute a mean to get distinct values per sensor
   getAllSensors() {
     return api.fetchNow().then(json => {
       let cells = _.chain(json)
+        .filter(json => json.node_moved == false)
         .map((value, key) => {
           let id = function(id) {
             for (let i in value.stats) {
-              return Number(value.stats[i].sensor_id);
+              if (value.node_moved == false) {
+                return Number(value.stats[i].sensor_id);
+              } else return null;
             }
           };
+          console.log(id());
           let lat = Number(value.location.latitude);
           let long = Number(value.location.longitude);
+          let location = value.location.name;
           let date = new Date(value.last_data_received_at);
-          let sensorsMoved = Boolean(value.sensors_moved);
+          //let sensorsMoved = Boolean(value.sensors_moved);
```

Comment:
If your removing it, just remove it instead of commenting it out.
Same [here](https://github.com/CodeForAfrica/sensors.AFRICA-AQ-map/pull/13/files#diff-3c9132b84184fb2fe1dc7c17226145dfR43)

Reply:
Done